### PR TITLE
[ADD] accounting: new doc about EU intra-community distance selling

### PR DESCRIPTION
**Pull request by jcs-odoo**
task-2582383
https://www.odoo.com/web#id=2582383&action=333&active_id=3835&model=project.task&view_type=form&cids=1&menu_id=4720

### CODE REVIEW

File: content/applications/finance/accounting/taxation/taxes/eu_distance_selling.rst [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+   Accounting settings.
+
+Fiscal Positions and Taxes
+==========================
+
+Once enabled, the feature creates all the necessary taxes and fiscal positions needed for each EU
+member state, based on your company's country.
+
+We highly recommend you check that the proposed mapping is suitable for the products and services
+you sell before using it.
```
Comment (vpk-odoo) [inline content/applications/finance/accounting/taxation/taxes/eu_distance_selling.rst:47]:
```suggestion
We highly recommend checking that the proposed mapping is suitable for the products and services
you sell before using it.
```

File: content/applications/finance/accounting/taxation/taxes/eu_distance_selling.rst [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+=============
+
+Go to :menuselection:`Accounting/Invoicing --> Settings --> Taxes`, then enable **EU intra-community
+distance selling** (or **EU Digital Goods VAT** if you created your database before July 1, 2021),
+and *Save*.
+
+.. important::
+   Please :ref:`upgrade the module <general/upgrade>` `l10n_eu_service` if you already installed it
+   before **July 1, 2021**, or if you activated the feature **EU Digital Goods VAT** in the
+   Accounting settings.
```
Comment (Yenthe666) [inline content/applications/finance/accounting/taxation/taxes/eu_distance_selling.rst:38]: Perhaps also a short mention that it means you need to update your Odoo instance it's source code if you are on premise too? I think quite some people wouldn't know that. You can link to the original docs about updates/deployments.

File: content/applications/finance/accounting/taxation/taxes/eu_distance_selling.rst [hunk 3]
```diff
@@ -0,0 +1,51 @@
+===================================
+EU intra-community distance selling
+===================================
+
+**Distance selling within the European Union** implies that organizations that sell goods and
+services cross-border to a private consumer (B2C) in an EU Member State must ensure that the VAT is
+paid to the Member State in which the goods or services are delivered.
```
[the remark this answers is not in the document]
Comment (jcs-odoo) [reply content/applications/finance/accounting/taxation/taxes/eu_distance_selling.rst:7]: Is this better?
It is also more detailed.

```
**Distance sales within the European Union** include cross-border sales of goods and services to a
private consumer (B2C) in another EU Member State when the seller doesn't meet face-to-face with the
customer. Organizations must ensure that the VAT on distance sales is paid to the Member State in
which the goods or services are delivered.
```

File: content/applications/finance/accounting/taxation/taxes/eu_distance_selling.rst [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+
+.. note::
+   This is also valid if your organization is located outside of the European Union.
+
+While this regulation mainly applies to :doc:`eCommerce <../../../../websites/ecommerce>` sales to
+private EU consumers, it is also valid for regular cross-border sales.
```
[the remark this answers is not in the document]
Comment (jcs-odoo) [reply content/applications/finance/accounting/taxation/taxes/eu_distance_selling.rst:13]:
```
While this regulation mainly applies to :doc:`eCommerce <../../../../websites/ecommerce>` sales to
private EU consumers, it is also valid for mail order sales and telesales.
```

File: content/applications/finance/accounting/taxation/taxes/eu_distance_selling.rst [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+
+The Union **One-Stop Shop (OSS)** is an online portal that businesses can use to register for the
+OSS and declare their transactions.
```
[the remark this answers is not in the document]
Comment (jcs-odoo) [reply content/applications/finance/accounting/taxation/taxes/eu_distance_selling.rst:16]:
```suggestion
The Union **One-Stop Shop (OSS)** is an online portal where businesses can register for the OSS and
declare their intra-community distance sales.
```

File: content/applications/finance/accounting/taxation/taxes/eu_distance_selling.rst (hunk 2 above)
[the remark this answers is not in the document]
Comment (jcs-odoo) [reply content/applications/finance/accounting/taxation/taxes/eu_distance_selling.rst:38]: it simply means that the module must be upgraded at this date.

the only thing is that it has a different name in the accounting app if the database was created before July 1, 2021.

Comment (jcs-odoo) [reply content/applications/finance/accounting/taxation/taxes/eu_distance_selling.rst:38]: Thanks for the feedback @Yenthe666 

I'll probably add this in the "general" doc, in the "upgrade" section referenced here rather than explain it in this documentation. This way, we don't overload all docs that require an upgrade, but if people need more information, they'll find it in the "upgrade apps and modules" section.

Since it's a change in 13.0, I'll do it with another PR.

**Comment by jcs-odoo** [comment]
Thanks for the feedback @vpk-odoo !

Let me know if the new phrasings are good now.

File: content/applications/finance/accounting/taxation/taxes/eu_distance_selling.rst (hunk 2 above)
Comment (Yenthe666) [reply content/applications/finance/accounting/taxation/taxes/eu_distance_selling.rst:45]: Sounds good, thanks :)

File: content/applications/finance/accounting/taxation/taxes/eu_distance_selling.rst [hunk 6]
```diff
@@ -0,0 +1,58 @@
...
+
+**Distance sales within the European Union** include cross-border sales of goods and services to a
+private consumer (B2C) in another EU Member State when the seller doesn't meet face-to-face with the
+customer. Organizations must ensure that the VAT on distance sales is paid to the Member State in
+which the goods or services are delivered.
+
+.. note::
+   This remains true even if your organization is located outside of the European Union.
+
+While this regulation mainly applies to :doc:`eCommerce <../../../../websites/ecommerce>` sales to
```
Comment (Feyensv) [inline content/applications/finance/accounting/taxation/taxes/eu_distance_selling.rst:13]:
```suggestion
While this regulation mainly applies to :doc:`eCommerce <applications/websites/ecommerce>` sales to
```

Links can be clearer and stronger if they are relative from the root content directory :) (and will stay valid, even if you move the current file).

Comment (jcs-odoo) [reply content/applications/finance/accounting/taxation/taxes/eu_distance_selling.rst:13]: Well, that's what I wrote initially but then @AntoineVDV  advised otherwise (in a different pull request)

-> https://github.com/odoo/documentation/pull/1044#discussion_r658545638

Comment (Feyensv) [reply content/applications/finance/accounting/taxation/taxes/eu_distance_selling.rst:13]: Well, he validated the opposite in https://github.com/odoo/documentation/pull/966 :thinking:.
If I remember correctly, we concluded sthg like "If there is more than three "../", then the link could be changed to an absolute one" ^^.
In this case, an absolute link is IMHO stronger than a relative one, but @AntoineVDV can have the final say :sweat_smile:

Comment (Feyensv) [reply content/applications/finance/accounting/taxation/taxes/eu_distance_selling.rst:13]: Forget it, we will use relative links to use a clear logic :ok_hand: (After discussion with ANV ^^)

**Review by Feyensv — APPROVED** [review]

File: content/applications/finance/accounting/taxation/taxes/eu_distance_selling.rst (hunk 6 above)
Comment (AntoineVDV) [reply content/applications/finance/accounting/taxation/taxes/eu_distance_selling.rst:13]: To be clear, the logic would be: use an "absolute" path in the ref when you need to traverse the root anyway.

Although @Feyensv is right that our relative link may be less robust than an absolute one, this is no longer the case when the page you're editing is buried deep in the toctree and only has the first few nodes of its path in common with the page you're making a reference to. At this point, the slightest change in the ancestor nodes of the target page will break every single link.

In short, both approaches are valid but one works best in some cases and vice-versa. Nevertheless, we need a universal logic to handle cross-app references.

File: content/applications/finance/accounting/taxation/taxes/eu_distance_selling.rst (hunk 3 above)
[the remark this answers is not in the document]
Comment (vpk-odoo) [reply content/applications/finance/accounting/taxation/taxes/eu_distance_selling.rst]: Much better!

File: content/applications/finance/accounting/taxation/taxes/eu_distance_selling.rst (hunk 4 above)
[the remark this answers is not in the document]
Comment (vpk-odoo) [reply content/applications/finance/accounting/taxation/taxes/eu_distance_selling.rst]: Looks good to me!

**Comment by Yenthe666** [comment]
Good job all!
